EssentialsProtect/src/main/java/com/earth2me/essentials/protect/ProtectConfig.java

```diff
@@ -18,6 +18,7 @@
     prevent_fire_spread("protect.prevent.fire-spread", true),
     prevent_flint_fire("protect.prevent.flint-fire", false),
     prevent_lava_fire_spread("protect.prevent.lava-fire-spread", true),
+    disable_piston_change("protect.prevent.piston.change",true),
```

Comment:
This should not be set by default since this breaks vanilla behavior in an unexpected (and normally undesired) way.
```suggestion
    disable_piston_change("protect.prevent.piston-change", false),
```